CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ## 5.0.4
 
 - fix: End file at the right place with #ifdef #521
+- ref: Session values are unsigned #527
```

Comment:
`m`: Shouldn't this be in `## unreleased`

Reply:
good catch! thanks